config/manifests/bases/openstack-operator.clusterserviceversion.yaml

```diff
@@ -316,7 +317,7 @@ spec:
         displayName: Template
         path: octavia.template
       - description: OpenStackClient - Parameters related to the OpenStackClient
-        displayName: OpenStack Client
+        displayName: Open Stack Client
```

Comment:
```suggestion
        displayName: OpenStack Client
```

Reply:
Just fixed in my latest PR. NOTE: while this showed up on the GA release it was because that release was missing the operators.operatorframework.io/internal-objects which should have also been set to hide this altogether. It was related to the downstream bundle generation as we already had that set upstream.